packages/fuel-gauge/src/await-execution.test.ts

```diff
@@ -0,0 +1,97 @@
+import { launchNode } from '@fuel-ts/wallet/test-utils';
+import {
+  Provider,
+  WalletUnlocked,
+  randomBytes,
+  Wallet,
+  BaseAssetId,
+  FUEL_NETWORK_URL,
+} from 'fuels';
+
+/**
+ * @group node
+ */
+describe('await-execution', () => {
+  test('awaiting execution of a transaction on the provider works', async () => {
+    const { cleanup, ip, port } = await launchNode({
+      args: ['--poa-interval-period', '400ms'],
+    });
+    const nodeProvider = await Provider.create(`http://${ip}:${port}/graphql`);
+
+    const genesisWallet = new WalletUnlocked(
+      process.env.GENESIS_SECRET || randomBytes(32),
+      nodeProvider
+    );
+
+    const destination = Wallet.generate({ provider: nodeProvider });
+
+    const transfer = await genesisWallet.createTransfer(destination.address, 100, BaseAssetId, {
+      gasPrice: nodeProvider.getGasConfig().minGasPrice,
+      gasLimit: 10_000,
+    });
+
+    transfer.updateWitnessByOwner(
+      genesisWallet.address,
+      await genesisWallet.signTransaction(transfer)
+    );
+
+    const response = await nodeProvider.sendTransaction(transfer, { awaitExecution: true });
+
+    expect(response.gqlTransaction?.status?.type).toBe('SuccessStatus');
+
+    cleanup();
+  });
+
+  test.skip('transferring funds with awaitExecution works', async () => {
```

Comment:
Why are we skipping this test?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1615#discussion_r1450445767